services/lsst-dev.rst

```diff
@@ -256,6 +257,39 @@ Administrators may wish to note that the shared stack is automatically updated u
 
 .. _HTCondor pool: https://confluence.lsstcorp.org/display/DM/Orchestration
 
+.. _lsst-dev-testdata:
+
+Validation/Test Data Sets
+=========================
+
+As of early April 2019, there are two ``cron`` jobs that will update a set of validation data repositories and test data repositories.
+These updates will trigger in the 02:00 hour local at the LDF.
```

Comment:
“in the 02:00 hour local at the LDF” sounds clunky. Could we just say something like “overnight on the lsst-dev system”?